examples/electrum_backend.rs

```diff
@@ -0,0 +1,88 @@
+use std::str::FromStr;
+
+use bdk::bitcoin::Network;
+use bdk::bitcoin::util::bip32::ExtendedPrivKey;
+use bdk::blockchain::{ElectrumBlockchain, Blockchain};
+use bdk::database::MemoryDatabase;
+use bdk::template::Bip84;
+use bdk::wallet::export::{FullyNodedExport};
+use bdk::{KeychainKind, Wallet, SyncOptions};
+
+use bdk::electrum_client::Client;
+use bdk::wallet::AddressIndex;
+use bitcoin::util::bip32;
+
+pub mod utils;
+
+use crate::utils::tx::build_signed_tx;
+
+/// This will create a wallet from an xpriv and get the balance by connecting to an Electrum server.
+/// If enough amount is available, this will send a transaction to an address.
+/// Otherwise, this will display a wallet address to receive funds.
+///
+/// This can be run with `cargo run --features="electrum" --example electrum_backend` in the root folder.
```

Comment:
nit: can omit the quote 
```suggestion
/// This can be run with `cargo run --features=electrum --example electrum_backend` in the root folder.
```

Reply:
Done in 7a47022ed697e87cb3f978bdeb4aef0c90c39d33.